waku/v2/peermanager/peer_manager.go

```diff
@@ -106,6 +109,8 @@ func NewPeerManager(maxConnections int, maxPeers int, logger *zap.Logger) *PeerM
 		subRelayTopics:         make(map[string]*NodeTopicDetails),
 		maxPeers:               maxPeers,
 		wakuprotoToENRFieldMap: map[protocol.ID]WakuProtoInfo{},
+		timesource:             timesource,
+		rttCache:               NewRTTCache(timesource, logger),
```

Comment:
I am wondering if a timesource is not passed, should we not initialize this cache and not have fastest selection feature or control this by a config option? This maybe useful for someone who doesn't want constant pings done to peers.

Reply:
Especially for a serviceNode use-case, we may not want to start this RTT Cache itself.

Reply:
This is no longer an issue since there's no periodic ping being executed now.